catalogue/webapp/components/IIIFSearchWithin/IIIFSearchWithin.tsx

```diff
@@ -146,7 +146,7 @@ const IIIFSearchWithin: FunctionComponent<Props> = ({
           )}
           <SearchButtonWrapper>
             <ButtonSolid
-              size="large"
+              size="medium"
```

Comment:
This was using the "old" large (which was very similar to medium), so changing the size so it still fits within its container.

before
<img width="451" alt="Screenshot 2022-11-10 at 12 25 02" src="https://user-images.githubusercontent.com/110461050/201322052-7474f086-d828-4586-809c-059fb06b5947.png">

after
<img width="436" alt="Screenshot 2022-11-10 at 12 24 55" src="https://user-images.githubusercontent.com/110461050/201322125-7358822c-af7e-49c2-ace2-db04611918e6.png">